feat/css style to avoid code word break in tables (#22155)
backstage/backstage: .changeset/tender-flowers-collect.md

---
'@backstage/plugin-techdocs': minor
---

> code tag css style changed to avoid word break

Reviewer: ```suggestion
Updated the styling for <code> tags to avoid word break.
```

Author: suggested change applied. thanks
